src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/AbstractAuditlogiUnitTest.java

```diff
@@ -105,9 +116,22 @@ protected boolean validateJson(final String json) {
 
     protected AuditMessageRouter createMessageRouterComplianceEnabled(Settings settings) {
         AuditMessageRouter router = new AuditMessageRouter(settings, null, null, null);
-        if (router.isEnabled()) {
-            router.enableRoutes(settings);
-        }
+        router.enableRoutes(settings);
         return router;
     }
+
+    protected void updateAuditConfig(final Settings settings) throws Exception {
+        updateAuditConfig(AuditTestUtils.createAuditPayload(settings));
+    }
+
+    protected void updateAuditConfig(final String payload) throws Exception {
+        final boolean sendAdminCertificate = rh.sendAdminCertificate;
+        final String keystore = rh.keystore;
+        rh.sendAdminCertificate = true;
+        rh.keystore = "auditlog/kirk-keystore.jks";
```

Comment:
Why is it OK to use this file?

Reply:
This is an admin cert used only in the tests to update the configuration.

Reply:
It comes from "auditlog", what if another cert is used in a different unit test? If it is always the same, why it comes from "auditlog"?

Reply:
Current usage of tests & associated test resources seem to be done this way. Tests in auditlog package are referencing the test resources (certs & config) present in the associated resources/auditlog directory.

Reply:
OK, but similar issue applies to `AuditTestUtils.updateAuditConfig()` that is used by `AbstractRestApiUnitTest`.